src/views/simulation/revenir-plus-tard.vue

```diff
@@ -0,0 +1,110 @@
+<script setup lang="ts">
+import { useStore } from "@/stores/index.js"
+import { EventAction, EventCategory } from "@lib/enums/event.js"
+import StatisticsMixin from "@/mixins/statistics.js"
+import { useRoute } from "vue-router"
+import { computed, ref } from "vue"
+import router from "@/router"
+
+const store = useStore()
+const route = useRoute()
+const simulationId = computed(() => store.simulationId)
+const saved = ref(false)
+const saveSimulationAndShowLink = async () => {
+  try {
+    StatisticsMixin.methods.sendEventToMatomo(
+      EventCategory.General,
+      EventAction.TemporarySimulationSave,
+      route.path
+    )
+    store.setSaveSituationError("")
+    await store.save()
+    saved.value = true
+  } catch (error: any) {
+    store.setSaveSituationError(error.response?.data || error)
+    StatisticsMixin.methods.sendEventToMatomo(
+      EventCategory.General,
+      EventAction.ErreurSauvegardeSimulation,
+      route.path
+    )
+  }
+}
+const temporarySimulationAccessUrl = computed(() => {
+  return `${process.env.VITE_BASE_URL}/simulation/recapitulatif?simulationId=${simulationId.value}`
```

Comment:
Pour moi il est nécessaire de créer une simulation et un followup pour avoir un lien partageable. En effet, ce lien là n'est pas utilisable depuis un autre navigateur/device.

Reply:
Oui, absolument, j'avais fait cette première itération "rapidement" surtout pour avoir vos premiers avis sur l'apect UI/UX de cette feature

Reply:
Oui, absolument, j'avais fait cette première itération rapidement surtout pour avoir vos premiers avis sur l'apect UI/UX de cette feature